docs/en/observability/logs-stream.asciidoc

```diff
@@ -143,10 +143,220 @@ If you're not seeing your log files in {kib}, check the following in the `elasti
+----
+2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.
+----
+
+To extract the timestamp you need to:
+- <<logs-stream-ingest-pipeline>>
+- <<logs-stream-index-template>>
+- <<logs-stream-create-data-stream>>
+
+[discrete]
+[[logs-stream-ingest-pipeline]]
+==== Extract the timestamp with an ingest pipeline
+
+To extract the `@timestamp` field, use an ingest pipeline with a dissect processor and a date processor. Processors are executed in the order they are defined, so in the following example: 
+
+. The {ref}/dissect-processor.html[dissect processor] extracts a structured field from your unstructured log message based on a pattern you set. In the following example, it extracts the timestamp to the `timestamp` field.
+. The {ref}/date-processor.html[date processor] then takes the `timestamp` field extracted by the dissect processor and extracts it to a date field. By default, this date field is called `@timestamp`. The date processor can also set the timezone, change the target field, and change the output format of the timestamp.
+
+The following example creates an ingest pipeline with a dissect and a date processor:
+
+[source,console]
+----
+PUT _ingest/pipeline/logs-generic-default
+{
+  "description": "Extracts the timestamp from log",
+  "processors": [
+    {
+      "dissect": {
+        "field": "message",
+        "pattern": "%{timestamp} %{message}"
+      }
+    },
+    {
+      "date": {
+        "field": "timestamp",
+        "formats": ["ISO8601"]
+      }
+    }
+  ]
+}
+----
+
+. Set the following values for the dissect processor:
+- `field` – The field you're extracting data from, `message` in this case.
+- `pattern`– The pattern of the elements in your log data:
++
+[source,JSON]
+----
+%{timestamp} %{message}
+----
++
+This extracts the timestamp, `2023-08-08T13:45:12.123Z`, to the `timestamp` field, and the rest of the message, `WARN 192.168.1.101 Disk usage exceeds 90%.`, stays in the `message` field.
++
+. Set the following for the date processor:
+- `_ingest/pipeline/logs-generic-default` – `logs-generic-default` is the name of the pipeline. The name of your pipeline needs to match the name of your data stream. You'll set up your data stream in the next section. See {fleet-ref}/data-streams.html#data-streams-naming-scheme[data stream naming scheme] for more information.
+- `field` – The field you're extracting the date from, `@timestamp` in this case.
+- `formats` – The format of the current timestamp, `ISO8601` in this case. The date processor accepts {ref}/mapping-date-format.html[Java time patterns] or the following formats: `ISO8601`, `UNIX`, `UNIX_MS`, `TAI64N`.
+
+Optionally, you can set the following values for the `date` processor:
+
+- `timezone` – Sets the timezone to use when parsing the date.
+- `target_field` – Sets the output field for the timestamp.
+- `output_format` – Changes the format when writing the date to the timestamp field. Must be a valid java time pattern.
+
+[discrete]
```

Comment:
before we create an index template, we can use the simulate API so the user sees direct results if it will work as expected and then hook up everything together: https://www.elastic.co/guide/en/elasticsearch/reference/current/simulate-pipeline-api.html#:~:text=You%20can%20use%20the%20simulate,verbose%20parameter%20to%20the%20request.